example-application/test/basic-tests.test.js

```diff
@@ -2,7 +2,7 @@ const request = require("supertest");
 const sinon = require("sinon");
 const nock = require("nock");
 const { initializeWebServer, stopWebServer } = require("../api-under-test");
-const mailer = require("../libraries/mailer");
+const mailer = require("../error-handling").mailer;
```

Comment:
1. how is this working? you don't export `mailer` in the `error-handling.js` file
2. This is weird that we need to import the mailer from `error-handling` file, what's wrong with what we have?

Reply:
It was a mistake, we don't even use this, Fixed.